tensorflow/lite/micro/kernels/batch_matmul.cc

```diff
@@ -271,22 +204,14 @@ TfLiteStatus TransposeRowsColumns(const TfLiteEvalTensor& tensor_in,
   return kTfLiteError;
```

Comment:
`TransposeRowsColumns` and `TransposeRowsColumnsImpl` is moved to the header file right? We should de-dupe from here?

Reply:
Nice catch, should be fixed now 😄